modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/helper/CPContentHelperImpl.java

```diff
@@ -480,6 +490,58 @@ public String getImageURL(FileEntry fileEntry, ThemeDisplay themeDisplay)
 		return cpMedia.getURL();
 	}
 
+	@Override
+	public String getIncomingQuantityLabel(
+			CommerceInventoryReplenishmentItem
+				commerceInventoryReplenishmentItem,
+			HttpServletRequest httpServletRequest)
+		throws PortalException {
+
+		User user = _portal.getUser(httpServletRequest);
+
+		Format dateFormatDateTime =
+			FastDateFormatFactoryUtil.getSimpleDateFormat(
```

Comment:
Can we use `getDate` instead without defining a pattern?

Reply:
getDate returns for example dd/MM/yyyy but the desing wants MMMM dd, yyyy

Reply:
It depends on the user locale